packages/rocketchat-ui-admin/client/admin.js

```diff
@@ -566,14 +568,16 @@ Template.admin.onRendered(function() {
 		SideNav.openFlex();
 	});
 	Tracker.autorun(function() {
-		const hasColor = TempSettings.findOne({
+		const hasColor = TempSettings.find({
 			group: FlowRouter.getParam('group'),
 			type: 'color'
-		}, { fields: { _id: 1 }});
+		}, { fields: { _id: 1, editor: 1 }}).fetch().length;
```

Comment:
if you only want to check if there is a record matching the filter, you can remove the `fields` and just `.count()` instead of `.fetch().length`

Reply:
I don't, I want to rerun the code to active the `jscolors` when missin